docs/open-api/lineage.yaml

```diff
@@ -0,0 +1,281 @@
+      description: An Input Dataset Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+    InputDataset:
+      description: An input dataset
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Dataset'
+        - type: object
+          properties:
+            inputFacets:
+              description: The input facets for this dataset.
+              type: object
+              anyOf:
+                - type: object
+                  additionalProperties:
+                    $ref: '#/components/schemas/InputDatasetFacet'
+    OutputDatasetFacet:
+      description: An Output Dataset Facet
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/BaseFacet'
+    OutputDataset:
+      description: An output dataset
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Dataset'
+        - type: object
+          properties:
+            outputFacets:
+              description: The output facets for this dataset
+              type: object
+              anyOf:
+                - type: object
+                  additionalProperties:
+                    $ref: '#/components/schemas/OutputDatasetFacet'
+    RunEvent:
+      allOf:
+        - $ref: '#/components/schemas/BaseEvent'
+        - type: object
+          properties:
+            eventType:
+              description: the current transition of the run state. It is required to issue 1 START event and 1 of [ COMPLETE, ABORT, FAIL ] event per run. Additional events with OTHER eventType can be added to the same run. For example to send additional metadata after the run is complete
+              type: string
+              enum:
+                - START
+                - RUNNING
+                - COMPLETE
+                - ABORT
+                - FAIL
+                - OTHER
+              example: START|RUNNING|COMPLETE|ABORT|FAIL|OTHER
+            run:
+              $ref: '#/components/schemas/Run'
+            job:
+              $ref: '#/components/schemas/Job'
+            inputs:
+              description: The set of **input** datasets.
+              type: array
+              items:
+                $ref: '#/components/schemas/InputDataset'
+            outputs:
+              description: The set of **output** datasets.
+              type: array
+              items:
+                $ref: '#/components/schemas/OutputDataset'
+          required:
+            - run
+            - job
+    StaticDataset:
+      description: A Dataset sent within static metadata events
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/Dataset'
+    DatasetEvent:
+      allOf:
+        - $ref: '#/components/schemas/BaseEvent'
+        - type: object
+          properties:
+            dataset:
+              $ref: '#/components/schemas/StaticDataset'
+          required:
+            - dataset
+          not:
+            required:
+              - job
+              - run
```

Comment:
what's this?

Reply:
`job` and `run` is not an requried properties, I guess there maybe some historical reasons